dev-python/nuitka/nuitka-0.6.6.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} )
+inherit distutils-r1 eutils
+
+DESCRIPTION="Python to native compiler"
+HOMEPAGE="https://www.nuitka.net"
+SRC_URI="https://nuitka.net/releases/${P^}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="dev-python/appdirs[${PYTHON_USEDEP}]
+	>dev-util/scons-3.0.5[${PYTHON_USEDEP}]
+	dev-python/setuptools[${PYTHON_USEDEP}]"
+
+S="${WORKDIR}/${P^}"
+
+pkg_postinst() {
+	optfeature "support for stand-alone executables" app-admin/chrpath
+}
```

Comment:
I see upstream changelog mentions: 
```
Standalone: Added support for dill.

Standalone: Added support for scikit-image and skimage.

Standalone: Added support for weasyprint.

Standalone: Added support for dask.

Standalone: Added support for pendulum.

Standalone: Added support for pytz and pytzdata.
```

Does this mean anything? More `optfeature` perhaps?

Reply:
These are libraries which can be distributed with the application. They need explicit support in nuitka. 
According to the source code, [standalone still only needs chrpath.](https://github.com/Nuitka/Nuitka/blob/00748128bc6acecfcb92e0a4aeaeee0c34f266c7/nuitka/freezer/Standalone.py#L1364).